hub/__init__.py

```diff
@@ -1,8 +1,21 @@
+__pdoc__ = {
+    "core": False,
+    "api": False,
+    "cli": False,
+    "client": False,
+    "constants": False,
+    "integrations": False,
+    "tests": False,
+}
+
 from .api.dataset import Dataset
+from .api.tensor import Tensor
 from .api.load import load
 from .core.transform.transform import transform  # type: ignore
 from .util.bugout_reporter import hub_reporter
 
+__all__ = ["Dataset", "Tensor", "load", "transform", "__version__"]
+
 __version__ = "2.0a3"
```

Comment:
why 2.0a3 and not current version 2.0a7?

Reply:
@tatevikh hasn't been back-merged. This will auto-resolve when merged into release/2.0.